### ISSUE TITLE
style error while setting width in case of rowSpan and colSpan

### ISSUE THREAD

**Issue by shalinibajpai0310** [AllenFang/react-bootstrap-table#1067]
Hi,
I am building a simple bootstrap table but getting the following error
 "Uncaught TypeError: Cannot read property 'style' of undefined".

I am getting this error whenever i am setting the width of the last row.
Here is my error code:

![image](https://cloud.githubusercontent.com/assets/20796527/23137278/20a7a69e-f7c7-11e6-90a0-8582d818b92d.png)

`

Here is my Working Code

![image](https://cloud.githubusercontent.com/assets/20796527/23137314/5d20ff26-f7c7-11e6-84a7-27db3960f21a.png)

I need to assign  width for the last row.
I am using Chrome with react: 15.4.1 and react-bootstrap-table: 2.11.2,
![errortable](https://cloud.githubusercontent.com/assets/20796527/23136808/7510704c-f7c4-11e6-82db-5f70f36e8ec0.PNG)
![tablegrp](https://cloud.githubusercontent.com/assets/20796527/23136812/7c5da824-f7c4-11e6-9e68-cdc0cb5c1e20.PNG)

  Can you help me with this..??

**Comment by AllenFang** [comment]
@shalinibajpai0310, It's seems like you forgot to include the css of `react-bootstrap-table`, check [this](https://github.com/AllenFang/react-bootstrap-table#cimport-css)

let me know if the problems still remain after you include this css

**Comment by shalinibajpai0310** [comment]
@Allen , i have included the css , still found the same issue.

Here is my index.html
![image](https://cloud.githubusercontent.com/assets/20796527/23196312/cd4521e8-f8de-11e6-85ca-e9e09314a45a.png)

this.props.children is having 9 objects whereas the header[] is having only  8 nodelist due to which it fails at the last index always.
![image](https://cloud.githubusercontent.com/assets/20796527/23196389/566c4f14-f8df-11e6-8c9f-b4ca43c43375.png)

![image](https://cloud.githubusercontent.com/assets/20796527/23196408/6cd8575c-f8df-11e6-83bf-4e63c1d88bd9.png)

**Comment by AllenFang** [comment]
ok, I gonna check this out, thanks :)

**Comment by AllenFang** [comment]
@shalinibajpai0310, I still cant reproduce your issue, anything I missed?
<img width="1103" alt="2017-04-09 10 26 03" src="https://cloud.githubusercontent.com/assets/5351390/24838088/a17f2540-1d73-11e7-9428-6c11537f8bd7.png">

**Comment by kant2002** [comment]
I have same issue and provide small repro

**Comment by AllenFang** [comment]
@kant2002, ok and you pass empty data into table?

**Comment by kant2002** [comment]
@AllenFang yes, data comes from API call, and sometimes there no data returned. Maybe that's not best UX experience to have empty table, but so far that was easiest to me.

**Comment by AllenFang** [comment]
ok, I'll close your code and test it on local, hope it can be reproduced. BTW, any indication for running your application? Thanks

**Comment by kant2002** [comment]
Since don't have any advanced usage of react-bootstrap-table I just rip-off this particular branch of code. I try to debug, to give pull request, but unfortunately I don't have time to fully understand how RBT works, so I give up on that.

**Comment by AllenFang** [comment]
thats ok

**Comment by kant2002** [comment]
@allenfang did you able take a look at the issue? Maybe you could give some even vague guidance why is that. I could send PR then

**Comment by AllenFang** [comment]
@kant2002, actually, I try your code in my local but still cant reproduce it.

**Comment by kant2002** [comment]
I think I manage to produce fix for this in https://github.com/AllenFang/react-bootstrap-table/pull/1318

**Comment by AllenFang** [comment]
thanks @kant2002's contribution again 👍   I'm not sure what's the problems you have but are you sure the #1318 can fix this issue? because I think #1318 is only fixing the `null` or `undefined` case for `TableHeaderColumn`, is that related with this issue?

**Comment by kant2002** [comment]
I think it only partially fix issue. I will double check my repro one more time. One corner case which #1318 does not fix is that there simple literals like whitespace occur inside of the `<BootstrapTable>` then this particular issue could occurs.
